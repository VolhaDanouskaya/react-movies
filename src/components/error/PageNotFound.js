import React from 'react'

import { Container } from '@material-ui/core'

import Footer from '../footer/Footer'

import { Link } from 'react-router-dom'

/* eslint-disable import/no-absolute-path */
import logo from '/public/images/logo.png'
import './error.scss'

const PageNotFound = () => (
  <>
    <Container className="error-page" fixed>
      <img className="logo" src={logo} alt="" />
      <div className="error-content">
        <p className="error">Page Not Found</p>
        <Link to="/" className="home-link">
          Go Back TO Home
        </Link>
      </div>
    </Container>
    <Footer />
  </>
)

export default PageNotFound
