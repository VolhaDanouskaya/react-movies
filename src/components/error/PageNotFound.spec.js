import React from 'react';

import { render } from '@testing-library/react';

import PageNotFound from './PageNotFound';

describe('test pagenotfound component', () => {
  test('whole pagenotfound should match snapshot', () => {
    const { asFragment } = render(<PageNotFound />);
    expect(asFragment(<PageNotFound />)).toMatchSnapshot();
  });
});
