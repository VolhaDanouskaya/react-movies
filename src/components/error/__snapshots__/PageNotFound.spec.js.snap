// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test pagenotfound component whole pagenotfound should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root errorPage MuiContainer-fixed MuiContainer-maxWidthLg"
  >
    <img
      alt="logo"
      class="logo"
      src="/images/logo.png"
    />
    <div
      class="errorContent"
    >
      <p
        class="error"
      >
        Page Not Found
      </p>
      <a
        href="/"
      >
        Go Back To Home
      </a>
    </div>
  </div>
  <div
    class="MuiContainer-root footer MuiContainer-fixed MuiContainer-maxWidthLg"
  >
    <img
      alt="logo"
      class="logo-footer"
      src="/images/logo.png"
    />
  </div>
</DocumentFragment>
`;
